src/Microsoft.VisualStudio.Validation.Tests.Shared/RequiresTests.cs

```diff
@@ -19,4 +19,25 @@ public void Guid_ThrowsOnEmpty()
         // Make sure an argument exception is thrown when an empty guid is passed in.
         Assert.Throws<ArgumentException>(() => Requires.NotEmpty(Guid.Empty, "foo"));
     }
+
+    [Fact]
+    public void Argument_Bool_String_String()
+    {
+        Requires.Argument(true, "a", "b");
+        Assert.Throws<ArgumentException>("a", () => Requires.Argument(false, "a", "b"));
+    }
+
+    [Fact]
+    public void Range_Bool_String_String()
+    {
+        Requires.Range(true, "a");
```

Comment:
It's actually the same overload (default parameter scenario). But I've added an assert for it anyway.